pkg/debuginfo/debuginfo.go

```diff
@@ -101,47 +111,46 @@ func (di *DebugInfo) EnsureUploaded(ctx context.Context, objFiles []*objectfile.
 			if errors.Is(err, debuginfo.ErrDebugInfoAlreadyExists) {
 				// If already exists, we should mark as exists in cache!
 				di.existsCache.Put(buildID, struct{}{})
-				level.Debug(logger).Log("msg", "debug info already exists")
+				level.Debug(logger).Log("msg", "debug information has already been uploaded or exists in server")
 				continue
 			}
-			level.Error(logger).Log("msg", "failed to upload debug info", "err", err)
+			level.Error(logger).Log("msg", "failed to upload debug information", "err", err)
 			continue
 		}
-		level.Debug(logger).Log("msg", "debug info uploaded successfully")
+		level.Debug(logger).Log("msg", "debug information successfully uploaded successfully")
+	}
 
-		// Successfully uploaded, we can clean up.
-		// Cleanup the extracted debug info file.
-		if shouldCleanup {
-			if err := os.Remove(dbgInfoPath); err != nil {
-				if os.IsNotExist(err) {
-					di.debugInfoFileCache.Invalidate(buildID)
-					continue
-				}
-				level.Debug(logger).Log("msg", "failed to cleanup debug info", "err", err)
+	for _, c := range filesToCleanup {
+		if err := os.Remove(c.path); err != nil {
+			if os.IsNotExist(err) {
+				di.debugInfoFileCache.Invalidate(c.buildID)
 				continue
 			}
-			di.debugInfoFileCache.Invalidate(buildID)
+
+			level.Debug(di.logger).Log(
+				"msg", "failed to cleanup debug information",
+				"buildid", c.buildID, "path", c.path, "err", err,
```

Comment:
nit, if we break these onto multiple lines already, I'd break them per key-value-pair

Reply:
Nothing that needs updating for this PR. Just wanted to state my preference (feel free to ignore) :smile: